### DISCUSSION TITLE
Landing page with random set of recipes (number configurable?)

### DISCUSSION THREAD

**Discussion by jecorn** [mealie-recipes/mealie#2307] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Often, recipe managers (and paper-and-ink cookbooks) are used as inspiration for what to cook. The various sort options on the landing page (nightly) are still relatively static, and offer views that do not change much over time. This is good when (for example) going back to something that one is about to make and has recently viewed. But not as useful when planning new things to make from a recipe collection. The current "Random" button is a step towards inspiration, but is awkward when scanning through multiple recipes looking for something to make. 

**(Optional): Suggest A Solution**
In addition to the random button, I propose making an additional page containing a random set of recipes. The number of recipes could be user-configurable, but infinite scroll like the main page would work just as well. Filtering by category would be ideal. For example, "I'm trying to figure out what to make for dinner this week" vs "What dessert should I bring to the party?"

From an implementation point of view, this could be identical to the current main landing page in nightly, but with randomised order. An additional button might be added to re-randomize the page, but infinite scrolling would also satisfy the use case by eventually showing something interesting to make. The page would potentially be re-randomized on every visit.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature
(sorry, but my python experience is far more back-end oriented than UX)

**Comment by michael-genson** [comment]
You can order by random on the landing page now. Not sure if this was available when this discussion was opened:
![image](https://github.com/user-attachments/assets/a5f789f0-a5b6-47d1-9e90-69298c0c666d)
